vmdb/db/migrate/20150111161446_create_nodes.rb

```diff
@@ -3,7 +3,7 @@ def up
     create_table :container_nodes do |t|
       t.string    :ems_ref
       t.string    :name
-      t.timestamp  :creation_timestamp
+      t.timestamp  :creation_timestamp, :null => true
```

Comment:
It's only for `timestamps` (the one that adds `created_at`/`updated_at`). It was made because it's impossible for those columns to actually contain NULL (because we always set them), so it's cleaner (and saves storage) to define the constraint.

A previous attempt (https://github.com/rails/rails/pull/3334) had to be reverted because it was an immediately breaking change. People still wanted it (e.g., https://github.com/rails/rails/pull/13885), so we found a way to do it through a deprecation cycle.

(Yes, versioned migration API would've made a much neater solution; this change was one of the main prompts that started us considering that for the future.)